index.html

```diff
@@ -0,0 +1,175 @@
+
+
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>PresiMon' | Battle!</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<div class="full-page clear-fix">
+		<h1 class="page-title">PresiMon' Pick your President and Battle!</h1>
+			<article>
+				<div class="card">
+					<header id="header" class="card-title">
+						<h2>Donald Trump</h2>
+					</header><!-- /header -->
+						<section class="card-product">
+							<img src="TrumpBattle.jpeg" alt="card-one-product" class="pic">
```

Comment:
Images are strange rangers. They take on properties of both block and inline elements, but they are not truly block. You can fix this by wrapping the image in a `<div>`.